general/management/commands/report_index_statuses.py

```diff
@@ -41,72 +44,82 @@ def add_arguments(self, parser):
                  'be modified.')
 
     def handle(self,  *args, **options):
+        self.log_start()
 
         # Get all solr ids
-        print "Getting solr ids...",
+        console_logger.info("Getting solr ids...")
         solr_ids = get_all_sound_ids_from_solr()
-        print "done!"
 
         # Get ell gaia ids
-        print "Getting gaia ids...",
+        console_logger.info("Getting gaia ids...")
         gaia_ids = Similarity.get_all_sound_ids()
-        print "done!"
 
-        print "Getting freesound db data..."
+        console_logger.info("Getting freesound db data...")
         # Get all moderated and processed sound ids
         queryset = Sound.objects.filter(processing_state='OK', moderation_state='OK').order_by('id').only("id")
         fs_mp = [sound.id for sound in queryset]
         # Get ell moderated, processed and analysed sounds
         queryset = Sound.objects.filter(processing_state='OK', moderation_state='OK', analysis_state='OK')\
             .order_by('id').only("id")
         fs_mpa = [sound.id for sound in queryset]
-        print "done!"
 
-        print "\nNumber of sounds per index:\n--------------------------"
-        print "Solr index\t\t%i" % len(solr_ids)
-        print "Gaia index\t\t%i" % len(gaia_ids)
-        print "Freesound\t\t%i  (moderated and processed)" % len(fs_mp)
-        print "Freesound\t\t%i  (moderated, processed and analyzed)" % len(fs_mpa)
-
-        print "\n\n***************\nSOLR INDEX\n***************\n"
         in_solr_not_in_fs = list(set(solr_ids).intersection(set(set(solr_ids).difference(fs_mp))))
         in_fs_not_in_solr = list(set(fs_mp).intersection(set(set(fs_mp).difference(solr_ids))))
-        print "Sounds in solr but not in fs:\t%i" % len(in_solr_not_in_fs)
-        print "Sounds in fs but not in solr:\t%i" % len(in_fs_not_in_solr)
+
+        message = "\nNumber of sounds per index:\n--------------------------"
+        message += "Solr index\t\t%i" % len(solr_ids)
```

Comment:
missing newlines after these messages. You could add them to a list and then do "\n".join(thelist)

Reply:
Addressed in 361ab791b6e78c01d165192a8c9ac6f8545f9217